src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java

```diff
@@ -64,13 +66,18 @@ public String getText() {
       return emptyText;
     }
 
-    if (forSecurityHotspot) {
-      return String.format(FORMAT, findings, findings == 1 ? "Security Hotspot" : "Security Hotspots", files, files == 1 ? "file" : "files");
+    String sinceText = "";
+    String newOrOldOrNothing = "";
```

Comment:
Maybe we can have a better name here, how about focusText?

Reply:
Not sure the suggestion is clearer. Naming is hard... Let's revisit later